src/main/java/javaClasses/homework_5/Booking/Taxi.java

```diff
@@ -0,0 +1,34 @@
+package javaClasses.homework_5.Booking;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
+@Setter
+@Getter
+@AllArgsConstructor
+public class Taxi implements Bookable, Stoppable, TransportPrinter {
+    final String TYPE = "TAXI";
+    private String city;
+    private String street;
+    private String carSize;
+    private String stopStreet;
+    private int stopTime;
+
+    @Override
+    public String bookTicket() {
+        return "Taxi";
+    }
+
+    @Override
+    public String makingStop() {
+        return stopStreet + " " + "street" + " " + stopTime + " " + "minute";
```

Comment:
As part of task makingStop method was default, so it doesn't need to be implemented in derived (child) class

Reply:
I have removed this overriding method.